__tests__/modules.ts

```diff
@@ -0,0 +1,108 @@
+import {test, expect} from '@jest/globals'
+
+import transform from '../helpers/transform'
+
+test('static imports', () => {
+  const code = `
+    import defaultExport, {namedExport} from 'foo'
+    import * as namespace from 'bar'
+    console.log(defaultExport, namedExport, namespace)
+  `
+
+  expect(transform({code, env: 'development'})).toMatchInlineSnapshot(`
+    import defaultExport, {namedExport} from 'foo'
+    import * as namespace from 'bar'
+    console.log(defaultExport, namedExport, namespace)
+  `)
+
+  expect(transform({code, env: 'production'})).toMatchInlineSnapshot(`
+    import defaultExport, {namedExport} from 'foo'
+    import * as namespace from 'bar'
+    console.log(defaultExport, namedExport, namespace)
+  `)
```

Comment:
I put up https://github.com/kensho-technologies/babel-preset/pull/118 as a sketch for how we could solve this kind of duplication.

Reply:
I think the grouping in https://github.com/kensho-technologies/babel-preset/pull/118 is worth it. To avoid forgetting to add the right environments every time and for more readability and insight into the outputs.